### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by SpiderDan98**
- Add unit tests

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,21 +14,31 @@
   ],
   "require": {
     "php": ">=7.1",
-    "fond-of-spryker/company-type": "dev-master"
+    "fond-of-spryker/company": "dev-master",
+    "fond-of-spryker/company-business-unit": "dev-master",
+    "fond-of-spryker/company-type": "dev-master",
+    "spryker/company-role": "^1.0.0"
```
Comment (daniel-rose) [inline composer.json]: @SpiderDan98 did you consider all possible versions?

File: src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Business/Model/CompanyUser.php [hunk 2]
```diff
@@ -53,6 +53,20 @@ class CompanyUser implements CompanyUserInterface
...
+     * @param \FondOfSpryker\Zed\CompanyUserCompanyAssigner\Persistence\CompanyUserCompanyAssignerRepository $companyUserCompanyAssignerRepository
+     * @param \Spryker\Zed\CompanyUser\Business\CompanyUserFacadeInterface $companyUserFacade
+     * @param \FondOfSpryker\Zed\Company\Business\CompanyFacadeInterface $companyFacade
+     * @param \FondOfSpryker\Zed\CompanyType\Business\CompanyTypeFacadeInterface $companyTypeFacade
+     * @param \Spryker\Zed\CompanyRole\Business\CompanyRoleFacadeInterface $companyRoleFacade
+     * @param \Spryker\Zed\CompanyBusinessUnit\Business\CompanyBusinessUnitFacadeInterface $companyBusinessUnitFacade
+     */
     public function __construct(
         CompanyUserCompanyAssignerConfig $companyUserCompanyAssignerConfig,
         CompanyUserCompanyAssignerRepository $companyUserCompanyAssignerRepository,
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Business/Model/CompanyUser.php]: @SpiderDan98 use the interface of repository.

**Review by daniel-rose — CHANGES_REQUESTED** [review]
@gengjozsef please review this pr too.

File: composer.json (hunk 1 above)
Comment (SpiderDan98) [reply composer.json]: @daniel-rose Yes now

File: src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Business/Model/CompanyUser.php (hunk 2 above)
Comment (SpiderDan98) [reply src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Business/Model/CompanyUser.php]: @daniel-rose I changed it and updated the tests

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**